core/src/main/java/org/infinispan/interceptors/InterceptorChain.java

```diff
@@ -229,17 +229,12 @@ public boolean addInterceptorAfter(CommandInterceptor toAdd, Class<? extends Com
       }
    }
 
-   public boolean addInterceptorBefore(CommandInterceptor toAdd, Class<? extends CommandInterceptor> beforeInterceptor, boolean isCustom) {
```

Comment:
It's in a public package, so I'm assuming there is a way for the user to get a hold of an `InterceptorChain` instance... if not, it would be nice to move it to the impl package.